python/pyspark/sql/dataframe.pyi

```diff
@@ -237,13 +237,16 @@ class DataFrame(PandasMapOpsMixin, PandasConversionMixin):
         subset: Optional[List[str]] = ...,
     ) -> DataFrame: ...
     def approxQuantile(
-        self, col: str, probabilities: List[float], relativeError: float
+        self,
+        col: Union[str, Tuple[str, ...], List[str]],
+        probabilities: Union[List[float], Tuple[float]],
+        relativeError: Union[int, float]
```

Comment:
In general `int` objects are valid substitutions for `float`, so I'd probably avoid listing them explicitly, unless they're treated differently.